### PR TITLE
Forum page topics api

### PR DESCRIPTION
**Pull request by Bcastillo44**
- Related Issue (include '#'): #652

- Description of changes made:
Created connection topics from API to topics of interest on Forum page 

- Is the feature complete/bug resolved/etc..:

Not all topics are being pulled from API, need to resolve

### CODE REVIEW

File: api/community/fixtures/community.json [hunk 1]
```diff
@@ -44,7 +44,79 @@
...
+    }
+  },
+  {
+    "model": "community.topic",
+    "pk": 4,
+    "fields": {
+      "title": "Teens",
+      "description": "How to keep your teens occupied while you're away"
+    }
+  },
```
Comment (0xZakk) [inline api/community/fixtures/community.json:60]: These fixtures look good!

File: client/src/components/CarouselDots/CarouselDots.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from "react";
 import "./CarouselDots.css";
-import ReactDOM from "react-dom";
+// import ReactDOM from "react-dom";
```
Comment (0xZakk) [inline client/src/components/CarouselDots/CarouselDots.js]: Can you delete any commented out code?

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@Bcastillo44 It looks like a lot of these are from your other PR - can you make it so this PR only has the code relevant to the issue it's for? Each PR should only have the code for a single user story or issue

File: client/src/pages/ForumPage/ForumPage.js [hunk 3]
```diff
@@ -20,7 +23,30 @@ const handleClick = e => {
...
+  axios.get(`${BASE_URL}/topics`)
+    .then(response => {
+      this.setState({ 
+        topics: response.data
+      })
+    })
+    .catch(error => {
+      console.log('Error', error);
+    })
+}
```
Comment (0xZakk) [inline client/src/pages/ForumPage/ForumPage.js]: Can you put this in the appropriate service in the `services/` directory?

File: client/src/pages/ForumPage/ForumPage.js [hunk 4]
```diff
@@ -34,7 +60,7 @@ const ForumPage = props => {
         />
       </Banner>
       <div className="secondNav">
-        <a className="secondNav">
+        <a href className="secondNav">
```
Comment (0xZakk) [inline client/src/pages/ForumPage/ForumPage.js]: Do you need this href? Should it have a value?

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@Bcastillo44 Looks good! Just a few small changes

**Review by 0xZakk — CHANGES_REQUESTED** [review]
Can you fix this page? It's really important that you test your work before making a PR

Screen Shot 2020-07-14 at 1 50 56 PM

**Comment by secondaryfun** [comment]
Branch https://github.com/CotripperPlatform/CoTrip/pull/685 has these changes. This branch can be closed without merging.

**Pull request closed without merging**